service/key/key.go

```diff
@@ -35,6 +35,10 @@ func IPSANs(customObject certificatetpr.CustomObject) []string {
 	return customObject.Spec.IPSANs
 }
 
+func Organizations(customObject certificatetpr.CustomObject) []string {
+	return customObject.Spec.Organizations
+}
```

Comment:
Lets please do the append magic do in here. The organizations needed are the cluster component and the configured orgs in the tpo.